src/main/scala/github/gphat/censorinus/Client.scala

```diff
@@ -56,7 +58,12 @@ class Client(
     val task = new Runnable {
       def tick(): Unit = try {
         Option(queue.take).map({ metric =>
```

Comment:
this should be `.foreach` not `.map`. `foreach` signals a side effect. Map is intended for pure functions.